src/main/java/teammates/logic/core/FeedbackSessionsLogic.java

```diff
@@ -86,6 +86,8 @@
     private static final String ERROR_NON_EXISTENT_FS_UPDATE = String.format(ERROR_NON_EXISTENT_FS_STRING_FORMAT, "update");
     private static final String ERROR_NON_EXISTENT_FS_CHECK = String.format(ERROR_NON_EXISTENT_FS_STRING_FORMAT, "check");
     private static final String ERROR_NON_EXISTENT_FS_REMIND = String.format(ERROR_NON_EXISTENT_FS_STRING_FORMAT, "remind");
+    private static final String ERROR_NON_EXISTENT_FS_CONFIRM = String.format(ERROR_NON_EXISTENT_FS_STRING_FORMAT,
+                                                                "confirm submission");
```

Comment:
Might be better if everything starting from `String.format` is on the next line.